doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -61,14 +61,16 @@
     [#799](https://github.com/eclipse-iceoryx/iceoryx2/issues/799)
 * CMake no longer installs unusable binaries when FetchContent is used
     [#814](https://github.com/eclipse-iceoryx/iceoryx2/issues/814)
+* Fix linker error on linux
```

Comment:
If you have another reason to commit, could you change this to `Fix build and linker error on gcc 8 and 9`?